### DISCUSSION TITLE
(off topic)

### DISCUSSION THREAD

**Discussion by MejanH** [rehypejs/rehype#44] [Q&A] [answered]
I tried using HTML `<p>` tag with classes. that paragraph didn't even parse. Here is parser code:

```jsx
    const processedContent = await unified()
      .use(parse)
      .use(remark2rehype)
      .use(html)
      .process(matterResult.content)
```

**Comment by MejanH** [comment]
solved with [rehype-raw](https://github.com/rehypejs/rehype-raw). Thanks for the plugin.

**Comment by wooorm** [comment, marked as answer]
This is off topic. remark is a different thing than rehype. Our [support doc](https://github.com/rehypejs/.github/blob/main/support.md) describes how and where to ask questions!
